detection_rules/misc.py

```diff
@@ -16,6 +16,12 @@
 
 _CONFIG = {}
 
+LICENSE_HEADER = """
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+""".strip()
+
```

Comment:
Thoughts on these tweaks?

```suggestion
LICENSE_HEADER = """
Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
or more contributor license agreements. Licensed under the Elastic License;
you may not use this file except in compliance with the Elastic License.
""".strip()

LICENSE_LINES = LICENSE_HEADER.splitlines()

PYTHON_LICENSE = "\n".join("# " + line for line in LICENSE_LINES)
JS_LICENSE = f"""
/*
{'\n'.join('* ' + line for line in LICENSE_LINES)}
 */
""".strip()
```

Reply:
++

This definitely cleans it up